src/helpers/validateString/index.js

```diff
@@ -0,0 +1,30 @@
+// // @flow
+const MIN_LENGTH = 2
+const MAX_LENGTH = 30
+
+type StringValidationOptions = {
+  type: string,
+  minLength?: number,
+  maxLength?: number
+}
+
+type StringValidationResult = {
+  isValid: boolean,
+  error: string
+}
+
+const validateString = ({
+  type,
+  minLength = MIN_LENGTH,
+  maxLength = MAX_LENGTH
+}: StringValidationOptions) => (string: string): StringValidationResult => {
```

Comment:
Idk how to properly type curried functions using Flow so I’m only typing the end result